src/main/scala/com/gu/rcspollerlambda/services/Kinesis.scala

```diff
@@ -16,11 +18,23 @@ object Kinesis extends Logging {
     rcsUpdates.map(publish).sequence_
   }
 
+  private val compressionMarkerByte: Byte = 0x00.toByte
+
+  private def compress(bytes: Array[Byte]): Array[Byte] = {
```

Comment:
Lifted straight out of [Grid](https://github.com/guardian/grid/blob/fdb58fe15b83322013909b0fd97e09c785b5fd6e/common-lib/src/main/scala/com/gu/mediaservice/lib/json/JsonByteArrayUtil.scala#L14-L22).